File: app/src/main/java/com/soft/credit911/Login/Model/LoginResponse.java
package com.soft.credit911.Login.Model;

import com.google.gson.annotations.SerializedName;

public class LoginResponse{

	@SerializedName("data")
	private Data data;

	@SerializedName("message")
	private String message;

	@SerializedName("status")
	private String status;

	public Data getData(){
		return data;
	}

	public String getMessage(){
		return message;
	}

	public String getStatus(){
		return status;
	}

	public class Data{

		@SerializedName("user_avatar")
		private String userAvatar;

		@SerializedName("api_token")
		private String apiToken;

		@SerializedName("last_name")
		private String lastName;

		@SerializedName("created_at")
		private String createdAt;

		@SerializedName("email_verified_at")
		private String emailVerifiedAt;

		@SerializedName("token_2fa_expiry")
		private String token2faExpiry;

		@SerializedName("updated_at")
		private String updatedAt;

		@SerializedName("token_2fa")
		private String token2fa;

		@SerializedName("phone_number")
		private String phoneNumber;

		@SerializedName("id")
		private int id;

		@SerializedName("first_name")
		private String firstName;

		@SerializedName("email")
		private String email;

		@SerializedName("status")
		private String status;

		public String getUserAvatar(){
			return userAvatar;
		}

		public String getApiToken(){
			return apiToken;
		}

		public String getLastName(){
			return lastName;
		}

		public String getCreatedAt(){
			return createdAt;
		}

		public String getEmailVerifiedAt(){
			return emailVerifiedAt;
		}

		public String getToken2faExpiry(){
			return token2faExpiry;
		}

		public String getUpdatedAt(){
			return updatedAt;
		}

		public String getToken2fa(){
			return token2fa;
		}

		public String getPhoneNumber(){
			return phoneNumber;
		}

		public int getId(){
			return id;
		}

		public String getFirstName(){
			return firstName;
		}

		public String getEmail(){
			return email;
		}

		public String getStatus(){
			return status;
		}
	}
}

File: app/src/main/java/com/soft/credit911/datamodel/LoginResponse.kt
package com.soft.credit911.datamodel

import com.google.gson.annotations.SerializedName

class LoginResponse {
    @SerializedName("data")
    var data: DataLoginObj? = null

    @SerializedName("message")
    val message: String? = null

    @SerializedName("status")
    val status: String? = null
}

     class DataLoginObj {
        @SerializedName("user_avatar")
        var userAvatar: String? = null

        @SerializedName("api_token")
        val apiToken: String? = null

        @SerializedName("last_name")
        var lastName: String? = null

        @SerializedName("created_at")
        val createdAt: String? = null

        @SerializedName("email_verified_at")
        val emailVerifiedAt: String? = null

        @SerializedName("token_2fa_expiry")
        val token2faExpiry: String? = null

        @SerializedName("updated_at")
        val updatedAt: String? = null

        @SerializedName("token_2fa")
        val token2fa: String? = null

        @SerializedName("phone_number")
        var phoneNumber: String? = null

        @SerializedName("id")
        val id = 0

        @SerializedName("first_name")
        var firstName: String? = null

        @SerializedName("email")
        val email: String? = null

        @SerializedName("status")
        val status: String? = null


        @SerializedName("postal_code")
        val postal_code: String? = null


        @SerializedName("state")
        val state: String? = null


        @SerializedName("city")
        val city: String? = null


        @SerializedName("address_line_2")
        val address_line_2: String? = null

        @SerializedName("address_line_1")
        val address_line_1: String? = null
    }
